openedx/adg/lms/utils/decorators.py

```diff
@@ -0,0 +1,26 @@
+"""
+A utility for custom decorators
+"""
+import functools
+
+from django.conf import settings
+from django.dispatch import receiver
+
+
+def suspendingreceiver(signal, **decorator_kwargs):
+    """
+    A custom decorator tests, which can suspend receiver by overriding the SUSPEND_RECEIVERS
+    setting per test method or class.
+    """
+
+    def our_wrapper(func):
+        @receiver(signal, **decorator_kwargs)
+        @functools.wraps(func)
+        def fake_receiver(sender, **kwargs):
```

Comment:
Can we please give it some specific name according to the use case

Reply:
I think this name is what it suggests. It is actually a fake receiver that returns nothing but consume the signal.